src/app/COMPONENTS/task-details/task-details.component.html

```diff
@@ -21,26 +21,31 @@
             </mat-form-field>
             <mat-form-field appearance="fill">
                 <mat-label>{{task.type}}</mat-label>
-                <input matInput formControlName="type" required>
-                <mat-error>
-                    Type is required.
-                </mat-error>
+                <mat-select formControlName="type">
+                    <mat-option value="Personal">Personal</mat-option>
+                    <mat-option value="Work">Work</mat-option>
+                </mat-select>
+                <mat-error>Type is required</mat-error>
             </mat-form-field>
             <mat-form-field appearance="fill">
                 <mat-label>{{task.status}}</mat-label>
-                <input matInput formControlName="status" required>
-                <mat-error>
-                    Status is required.
-                </mat-error>
+                <mat-select formControlName="status">
+                    <mat-option value="Personal">In progress</mat-option>
+                    <mat-option value="Awaiting approval">Awaiting approval</mat-option>
```

Comment:
usually the values for options and whatnot should be 1 word so i'd say something like AWAITING_APPROVAL